packages/openneuro-app/src/scripts/datalad/fragments/dataset-summary.jsx

```diff
@@ -68,7 +74,10 @@ export default class Summary extends React.PureComponent {
             <hr />
             <h5>
               <b>
-                {files}, {size}, {subjects}, {sessions}
+                {accessionNumber}
+                <h5>
```

Comment:
now that files, size, subjects, and sessions are separated out into their own h5 element, lets remove the h5 and b wrappers (lines 75 & 76)

Reply:
The others are `span` tags, but I removed the redundant bold.